be/src/cloud/cloud_engine_calc_delete_bitmap_task.cpp

```diff
@@ -314,11 +311,6 @@ Status CloudTabletCalcDeleteBitmapTask::_handle_rowset(
     }
 
     _engine_calc_delete_bitmap_task->add_succ_tablet_id(_tablet_id);
-    LOG(INFO) << "calculate delete bitmap successfully on tablet"
```

Comment:
I think if update_delete_bitmap_time_us> 5*1000, print this log is better, because if metaservice is busy, update delete bitmap may cost some time